pkg/operator/certrotation/signer.go

```diff
@@ -183,7 +183,7 @@ func getValidityFromAnnotations(annotations map[string]string) (notBefore time.T
 
 // setSigningCertKeyPairSecret creates a new signing cert/key pair and sets them in the secret
 func setSigningCertKeyPairSecret(signingCertKeyPairSecret *corev1.Secret, validity time.Duration) error {
-	signerName := fmt.Sprintf("%s_%s@%d", signingCertKeyPairSecret.Namespace, signingCertKeyPairSecret.Name, time.Now().Unix())
+	signerName := fmt.Sprintf("%s_%s@%d", signingCertKeyPairSecret.Namespace, signingCertKeyPairSecret.Name, time.Now().UnixNano())
```

Comment:
That seems to be required for tests only, right? I don't think its suitable for production though, as it may grow too large?

Reply:
yes, for tests only, if we want to make sure the annotation changes as expected, otherwise we have to add `sleep` to the test